src/RegisterWorkerApertures.jl

```diff
@@ -38,14 +38,21 @@ end
 function init!(algorithm::Apertures)
     if algorithm.dev >= 0
         cuda_init!(algorithm)
+        CUDA.allowscalar()
```

Comment:
@timholy , 
If CUDA.allowscalar(false) (CUDA doesn't allow scalar indexing in default), it cause 'LoadError: Scalar indexing is disallowed.' error at
(https://github.com/HolyLab/RegisterMismatch.jl/blob/8fcf86abcfaa3406bda5b9be21eaab638a6c9558/src/RegisterMismatch.jl#L290).
So, I added CUDA.allowscalar() (which is same as CUDA.allowscalar(true))   but I'm not sure this is right approach.